### PR TITLE
NFDIV-2383, 2384: Welsh withdraw screen add translation for sole and joint applications

### PR DESCRIPTION
**Pull request by FaisalMoJ**
### Change description ###

Add Welsh language translation for Sole and Joint applications for Withdraw your application screen. 

### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/NFDIV-2383
https://tools.hmcts.net/jira/browse/NFDIV-2384

**Before merging a pull request make sure that:**

- [ ] tests have been updated / new tests has been added (if needed)
- [ ] README and other documentation has been updated / added (if needed)

**If this ticket will have any visible impact on users and is not behind a feature toggle, make sure that:**
- [ ] this ticket been reviewed by QA
- [ ] the user story been signed off by the PO

### CODE REVIEW

File: src/main/steps/applicant1/withdrawing-your-application/content.ts [hunk 1]
```diff
@@ -74,8 +74,77 @@ const en = ({ isDivorce, partner, isApplicant2 }: CommonContent) => ({
   exitLink: 'Exit service',
 });
 
-// @TODO translations
-const cy = en;
+const cy = ({ isDivorce, partner, isApplicant2 }: CommonContent) => ({
```
Comment (lara-mpt) [inline src/main/steps/applicant1/withdrawing-your-application/content.ts:77]:
```suggestion
const cy: typeof en = ({ isDivorce, partner, isApplicant2 }: CommonContent) => ({
```

File: src/main/steps/applicant1/withdrawing-your-application/content.ts [hunk 2]
```diff
@@ -74,8 +74,77 @@ const en = ({ isDivorce, partner, isApplicant2 }: CommonContent) => ({
...
+    line2:
+      'Os oes arnoch angen help gallwch gysylltu â’r llys gan ddefnyddio’r manylion isod. Ni all y staff cynorthwyol roi cyngor cyfreithiol i chi.',
+  },
+  joint: {
+    line1: {
+      part1: `Rydych wedi dweud nad ydych eisiau bwrw ymlaen â‘ch ${
+        isDivorce ? 'ysgariad' : 'cais i ddod â’ch partneriaeth sifil i ben'
+      }.
+      Os ydych eisiau tynnu eich cais yn ôl, yna mae angen i chi a’ch ${partner} lenwi `,
+      part2: 'D11 form',
```
Comment (lara-mpt) [inline src/main/steps/applicant1/withdrawing-your-application/content.ts:98]: should this be 'ffurflen D11' ?

**Review by lara-mpt — APPROVED** [review]

File: src/main/steps/applicant1/withdrawing-your-application/content.ts (hunk 1 above)
Comment (FaisalMoJ) [reply src/main/steps/applicant1/withdrawing-your-application/content.ts]: Added typeof

File: src/main/steps/applicant1/withdrawing-your-application/content.ts (hunk 2 above)
Comment (FaisalMoJ) [reply src/main/steps/applicant1/withdrawing-your-application/content.ts]: As suggested updated the English to Welsh

**Pull request merged**